data/tuyoData.json

```diff
@@ -15,9 +45,9 @@
     ],
     "tarashanaquotes": [
         {
-            "quote": "The first arrow missed only because the gods were kind.",
+            "quote": "\"This is not only bread,\" I said. \"It means something. What does it mean?\"",
```

Comment:
Looks like this quote ended up in the wrong place and accidentally deleted the Tarashana quote

Reply:
🤦‍♀️ So sorry, have no idea how I accidentally did that lol. Fixed in [896073f](https://github.com/RebeccaAnne/QuoteBot/pull/22/commits/896073fd8041be6d8f26a1a8c322fe046aeb5295)